flytekit/bin/entrypoint.py

```diff
@@ -445,14 +445,14 @@ def _pass_through():
 def execute_task_cmd(
     inputs,
     output_prefix,
-    raw_output_data_prefix,
-    test,
-    prev_checkpoint,
-    checkpoint_path,
-    dynamic_addl_distro,
-    dynamic_dest_dir,
-    resolver,
-    resolver_args,
+    raw_output_data_prefix=None,
```

Comment:
databricks will use a different entrypoint file uploaded to dbfs right?

Reply:
yes, sorry. I reverted it.